### ISSUE TITLE
Getting Deprecation Warnings for The Last Couple of Versions

### ISSUE THREAD

**Issue by 23d1** [craftcms/commerce#563]
### Description
Been getting deprecation warnings the last couple of versions. I believe it started with Beta 12.

Here's some info;
```
craft\commerce\elements\db\OrderQuery::updatedBefore is deprecated. Use dateUpdated() instead.
/httpd.www/craft/vendor/craftcms/commerce/src/elements/db/OrderQuery.php:136

craft\commerce\elements\db\OrderQuery::updatedAfter is deprecated. Use dateUpdated() instead.
/httpd.www/craft/vendor/craftcms/commerce/src/elements/db/OrderQuery.php:133
```

Stack trace 1;
```
Deprecation error: craft\commerce\elements\db\OrderQuery::updatedBefore is deprecated. Use dateUpdated() instead. Called from /httpd.www/craft/vendor/craftcms/commerce/src/elements/db/OrderQuery.php:393
craft\commerce\elements\db\OrderQuery::updatedBefore(DateTime) Called from /httpd.www/craft/vendor/craftcms/commerce/src/elements/db/OrderQuery.php:136
craft\commerce\elements\db\OrderQuery::__set("updatedBefore", DateTime) Called from /httpd.www/craft/vendor/yiisoft/yii2/BaseYii.php:546
yii\BaseYii::configure(craft\commerce\elements\db\OrderQuery, ["updatedBefore" => DateTime, "isCompleted" => "not 1"]) Called from /httpd.www/craft/vendor/craftcms/cms/src/controllers/ElementIndexesController.php:299
craft\commerce\controllers\ChartsController::_elementQuery() Called from /httpd.www/craft/vendor/craftcms/cms/src/controllers/ElementIndexesController.php:85
craft\commerce\controllers\ChartsController::init() Called from /httpd.www/craft/vendor/yiisoft/yii2/base/BaseObject.php:109
craft\commerce\controllers\ChartsController::__construct([]) Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Controller.php:86
craft\commerce\controllers\ChartsController::__construct("charts", craft\commerce\Plugin, [])
ReflectionClass::newInstanceArgs(["charts", craft\commerce\Plugin, []]) Called from /httpd.www/craft/vendor/yiisoft/yii2/di/Container.php:375
yii\di\Container::build("craft\commerce\controllers\ChartsController", ["charts", craft\commerce\Plugin], []) Called from /httpd.www/craft/vendor/yiisoft/yii2/di/Container.php:156
yii\di\Container::get("craft\commerce\controllers\ChartsController", ["charts", craft\commerce\Plugin]) Called from /httpd.www/craft/vendor/yiisoft/yii2/BaseYii.php:345
yii\BaseYii::createObject("craft\commerce\controllers\ChartsController", ["charts", craft\commerce\Plugin]) Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:642
craft\commerce\Plugin::createControllerByID("charts") Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:596
craft\commerce\Plugin::createController("get-revenue-data") Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:588
craft\web\Application::createController("charts/get-revenue-data") Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:522
craft\web\Application::runAction("commerce/charts/get-revenue-data", ["p" => "admin/actions/commerce/charts/get-revenue-data"]) Called from /httpd.www/craft/vendor/craftcms/cms/src/web/Application.php:282
craft\web\Application::runAction("commerce/charts/get-revenue-data", ["p" => "admin/actions/commerce/charts/get-revenue-data"]) Called from /httpd.www/craft/vendor/craftcms/cms/src/web/Application.php:542
craft\web\Application::_processActionRequest(craft\web\Request) Called from /httpd.www/craft/vendor/craftcms/cms/src/web/Application.php:266
craft\web\Application::handleRequest(craft\web\Request) Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Application.php:386
craft\web\Application::run() Called from /httpd.www/index.php:21
```

Stack trace 2;
```
Deprecation error: craft\commerce\elements\db\OrderQuery::updatedAfter is deprecated. Use dateUpdated() instead. Called from /httpd.www/craft/vendor/craftcms/commerce/src/elements/db/OrderQuery.php:372
craft\commerce\elements\db\OrderQuery::updatedAfter(DateTime) Called from /httpd.www/craft/vendor/craftcms/commerce/src/elements/db/OrderQuery.php:133
craft\commerce\elements\db\OrderQuery::__set("updatedAfter", DateTime) Called from /httpd.www/craft/vendor/yiisoft/yii2/BaseYii.php:546
yii\BaseYii::configure(craft\commerce\elements\db\OrderQuery, ["updatedAfter" => DateTime, "isCompleted" => "not 1"]) Called from /httpd.www/craft/vendor/craftcms/cms/src/controllers/ElementIndexesController.php:299
craft\commerce\controllers\ChartsController::_elementQuery() Called from /httpd.www/craft/vendor/craftcms/cms/src/controllers/ElementIndexesController.php:85
craft\commerce\controllers\ChartsController::init() Called from /httpd.www/craft/vendor/yiisoft/yii2/base/BaseObject.php:109
craft\commerce\controllers\ChartsController::__construct([]) Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Controller.php:86
craft\commerce\controllers\ChartsController::__construct("charts", craft\commerce\Plugin, [])
ReflectionClass::newInstanceArgs(["charts", craft\commerce\Plugin, []]) Called from /httpd.www/craft/vendor/yiisoft/yii2/di/Container.php:375
yii\di\Container::build("craft\commerce\controllers\ChartsController", ["charts", craft\commerce\Plugin], []) Called from /httpd.www/craft/vendor/yiisoft/yii2/di/Container.php:156
yii\di\Container::get("craft\commerce\controllers\ChartsController", ["charts", craft\commerce\Plugin]) Called from /httpd.www/craft/vendor/yiisoft/yii2/BaseYii.php:345
yii\BaseYii::createObject("craft\commerce\controllers\ChartsController", ["charts", craft\commerce\Plugin]) Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:642
craft\commerce\Plugin::createControllerByID("charts") Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:596
craft\commerce\Plugin::createController("get-revenue-data") Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:588
craft\web\Application::createController("charts/get-revenue-data") Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Module.php:522
craft\web\Application::runAction("commerce/charts/get-revenue-data", ["p" => "admin/actions/commerce/charts/get-revenue-data"]) Called from /httpd.www/craft/vendor/craftcms/cms/src/web/Application.php:282
craft\web\Application::runAction("commerce/charts/get-revenue-data", ["p" => "admin/actions/commerce/charts/get-revenue-data"]) Called from /httpd.www/craft/vendor/craftcms/cms/src/web/Application.php:542
craft\web\Application::_processActionRequest(craft\web\Request) Called from /httpd.www/craft/vendor/craftcms/cms/src/web/Application.php:266
craft\web\Application::handleRequest(craft\web\Request) Called from /httpd.www/craft/vendor/yiisoft/yii2/base/Application.php:386
craft\web\Application::run() Called from /httpd.www/index.php:21
```

### Additional info

- Craft version: 3.0.30.2
- PHP version: 7.2
- Database driver & version: MySQL/MariaDB
- Plugins & versions: Craft Commerce 2.0.0-beta.13.1

**Comment by lukeholder** [comment]
Thanks, we are aware and will remove the deprecation soon.

**Comment by lukeholder** [comment]
Fixed in d79dae3120dd62fccd18c44
